### PR TITLE
feat: Implement project settings/starred UI

### PR DESCRIPTION
**Pull request by lynnagara**
Required for environments feature

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by MaxBittker** [comment]
is it still easy to click the dropdown? i think it could be worth having padding for an enlarged clickable area because IMO the current one is too hard to click for how important it is

File: src/sentry/static/sentry/app/components/projectHeader/index.jsx [hunk 1]
```diff
@@ -46,39 +56,13 @@ class ProjectHeader extends React.Component {
             <li className={navSection == 'releases' ? 'active' : ''}>
               <Link to={`/${org.slug}/${project.slug}/releases/`}>{t('Releases')}</Link>
             </li>
+            {access.has('project:write') && (
+              <li className={navSection == 'settings' ? 'active' : ''}>
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/components/projectHeader/index.jsx]: maybe all three of these `li`s should use classNames?

**Review by MaxBittker — APPROVED** [review]
the code looks good to me besides my comments! thanks for removing the dead styles.

**Pull request merged**